apps/aehttp/test/aehttp_client.erl

```diff
@@ -14,22 +14,22 @@ request(OpId, Params, Cfg) ->
     BaseUrl = string:trim(proplists:get_value(Interface, Cfg), trailing, "/"),
     Path = endpoints:path(Method, OpId, Params),
     Query = endpoints:query(Method, OpId, Params),
-    inets:start(httpc, [{profile, test_browser}]),
-    Response = request(Method, BaseUrl, Path, Query, Params, [], [{timeout, 15000}], [], Cfg),
-    inets:stop(httpc, test_browser),
+    {ok, ClientPid} = inets:start(httpc, [{profile, test_browser}], stand_alone),
+    Response = request(Method, BaseUrl, Path, Query, Params, [], [{timeout, 15000}], [], ClientPid, Cfg),
+    ok = gen_server:stop(ClientPid, normal, infinity),
     Response.
 
-request(get, BaseUrl, Path, Query, _Params, Headers, HttpOpts, Opts, Cfg) ->
+request(get, BaseUrl, Path, Query, _Params, Headers, HttpOpts, Opts, Profile, Cfg) ->
```

Comment:
Is the `Profile` parameter always required now? is there no meaningful default value?
Couldn't it be shoved inside `Opts` and extracted using:
```
Profile = proplists:get_value(profile, Opts, test_browser),
Opts2 = lists:keydelete(profile, 1, Opts),
```

Reply:
> Is the Profile parameter always required now?

Yes. Though this is internal function so shall be ok.

> is there no meaningful default value?

Using a default value would hinder the benefits this commit adds - namely enhanced error handling and potential parallel clients. From commit message:

> * Linking httpc_manager process to test process - rather than to
>   httpc_profile_sup process;
> * Not registering httpc_manager process by (profile-based) name - so
>   to enable future concurrent clients;

I suggest to leave this as is and add default value if/when required.

> Couldn't it be shoved inside Opts ...

`Opts` is passed to `httpc:request` so I would prefer not to reuse it.